app_flutter/lib/commit_box.dart

```diff
@@ -115,11 +105,12 @@ class CommitOverlayContents extends StatelessWidget {
                 ListTile(
                   leading: CircleAvatar(
                     radius: 25.0,
-                    backgroundImage: NetworkImage(widget.avatarUrl),
+                    backgroundImage: NetworkImage(commit.authorAvatarUrl),
                     backgroundColor: Colors.transparent,
                   ),
-                  title: Text(widget.message),
-                  subtitle: Text(widget.author),
+                  // TODO(chillers): Show commit message here instead
```

Comment:
TODOs should have bugs filed and a link to the bug.  `// TODO(chillers): Show commit message here instead: https://github.com/flutter/flutter/issues/999999`

Reply:
Thank you Dan! I learned a new thing :)